### PR TITLE
Switch S3 configs from Fred Hutch to BBI AWS account (linelists, SCH, UW Retros)

### PR DESCRIPTION
**Pull request by kschwabefry**
We are migrating processes to the BBI AWS account as part
of consolidating the organization. Update the location of
various S3 files accordingly.

Migrate the configs for linelists, SCH and UW retros

### CODE REVIEW

**Review by davereinhart — CHANGES_REQUESTED** [review]
One other place where the FH S3 URL shows up: https://github.com/seattleflu/backoffice/blob/76a309c6270b6e8049141322b364d7b788f40ea2/bin/import-uw-retrospectives-to-redcap#L74

File: crontabs/wa-doh-linelists [hunk 1]
```diff
@@ -17,9 +17,9 @@ XDG_RUNTIME_DIR=/home/ubuntu/run
...
 FLOCK_DIR=/var/run/lock
 
-# SSH credentials for uploading to sft.wa.gov 
+# SSH credentials for uploading to sft.wa.gov
 DOH_USERNAME="doh-elff-NorthwestGenomicsCenterSCAN"
 DOH_PRIVKEY_PATH="/home/ubuntu/.ssh/sft_id_rsa"
 
 # Generate and upload the previous day's linelist data once a day at 6 A.M.
-0 6 * * * ubuntu promjob "wa-doh-linelists" chronic pipenv run envdir $ENVD/hutch envdir $ENVD/redcap /opt/backoffice/bin/wa-doh-linelists/generate --date $(date --date=yesterday --iso-8601=date) --output-dir "s3://fh-pi-bedford-t/seattleflu/public-health-reporting" --output-dir "ssh://sft.wa.gov" --upload-if-empty "s3://fh-pi-bedford-t/seattleflu/public-health-reporting"
+0 6 * * * ubuntu promjob "wa-doh-linelists" chronic pipenv run envdir $ENVD/hutch envdir $ENVD/redcap /opt/backoffice/bin/wa-doh-linelists/generate --date $(date --date=yesterday --iso-8601=date) --output-dir "s3://bbi-id3c-receiving/public-health-reporting" --output-dir "ssh://sft.wa.gov" --upload-if-empty "s3://bbi-id3c-receiving/public-health-reporting"
```
[the remark this answers is not in the document]
Comment (kschwabefry) [reply crontabs/wa-doh-linelists:25]: Good suggestion, I've just reated `env.d/aws-bbi` and added the new access and secret keys there.

**Review by davereinhart — APPROVED** [review]

**Comment by kschwabefry** [comment]
Deployed.

**Comment by kschwabefry** [comment]
[Transition FH S3 bucket to BBI Bucket](https://trello.com/c/LJIwCTP4/1175-transition-fh-s3-bucket-to-bbi-bucket)

**Pull request merged**